whist/server/api/game/create.py

```diff
@@ -46,23 +46,27 @@ def _set_game_parameter(request, user, pwd_service: PasswordService):
 def _get_amount_player(request, key) -> Optional[int]:
     if key not in ['min_player', 'max_player']:
         raise KeyError(f'{key} is not a valid key for this operation.')
-    if key in request:
+    try:
         return int(request[key])
-    return None
+    except (KeyError, TypeError):
```

Comment:
Error handling is one level above. Type should not be uncatched. The user tries to send the correct key, but made a mistake. We should inform them.